api/v1alpha1/queuedworkload_types.go

```diff
@@ -88,12 +88,15 @@ type PodSet struct {
 
 // QueuedWorkloadStatus defines the observed state of QueuedWorkload
 type QueuedWorkloadStatus struct {
-	// conditions hold the latest available observations of the QueuedWorkload
+	// Current hold the latest available observations of the QueuedWorkload
 	// current state.
 	// +optional
+	Current QueuedWorkloadCondition `json:"current,omitempty"`
```

Comment:
Do we need to add `current` separately？ User can get it from conditions, right?

Reply:
+1, splitting like this is against the pattern, conditions are already timestamped.